spec/phlex/component_spec.rb

```diff
@@ -178,6 +178,26 @@ def template
     end
   end
 
+  describe "with underscore in class name" do
+    let :component do
+      Class.new Phlex::Component do
+        def template
+          div class: "a_b"
+        end
+      end
+    end
+
+    it "when convert_underscores_to_dashes config is true convert underscores to dashes and produces the classes" do
+      expect(output).to eq %{<div class="a-b"></div>}
+    end
+
+    it "when convert_underscores_to_dashes config is false do not convert underscores to dashes and produces the classes" do
+      Phlex.configure { |config| config.convert_underscores_to_dashes = false }
```

Comment:
We should be able to put the configuration in an around each hook rather than include it in the `it`. https://relishapp.com/rspec/rspec-core/v/2-0/docs/hooks/around-hooks